src/rootPages/Designer/properties/process/ABProcessEnd.js

```diff
@@ -43,6 +43,7 @@ export default function (AB) {
                         id: ids.name,
                         view: "text",
                         label: L("Name"),
+                        labelWidth: 100,
```

Comment:
Should it be better if we use `AB.Config.labelWidthMedium` or `AB.Config.labelWidthLarge`  instead ?
https://github.com/digi-serve/ab_platform_web/blob/master/config/configDesktop.js#L46

Reply:
Ok, Thanks